scripts/users/update.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "{
+  \"email\": \"$USER_EMAIL\",
+  \"password\": \"$PASSWORD\"
+}" | $HTTP put "$IM_HOST/users/$1" "Authorization: Bearer $ACCESS_TOKEN"
+
```

Comment:
Do we really want all of this here?

Reply:
Do you mean the email and password JSON body for the request? I just went with the simplest example of updating both. 🤔